elevation-plot/journey-specs.js

```diff
@@ -0,0 +1,75 @@
+import { getRoutePath } from './queries.js';
+
+/**
+ * Show route path specific information like elevation, consumption, speed etc.
+ * The temperature and maxspeed are not yet part of the API. This will be added soon.
+ * @param path {object} route path specification.
+ */
+export const displaySpecs = path => {
+  // The consumption, in kWh, of this route path segment.
+  document.getElementById('consumption').innerHTML = path.routePath.consumptionPerKm * 1000 + ' Wh/km';
+  // The elevation (altitude) in meters for this route path segment.
+  document.getElementById('height').innerHTML = path.routePath.elevation + ' m';
```

Comment:
Just `height` as a name doesn't really say much, better to have more meaningful IDs, not only in JS, but also in HTML ;)